controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -73,6 +73,106 @@ static bool validate_hex_notation(const std::string &str, uint8_t expected_octet
     return true;
 }
 
+/**
+ * @brief Parse bss_info string into bss_info_conf_struct.
+ * 
+ * @param[in] bss_info_str String conataining bss info configuration
+ * @param[out] bss_info_conf Controller database struct filled with configuration.
+ * @param[out] err_string Contains an error description if the function fails.
+ * @return true if successful, false if not.
+ */
+static std::string parse_bss_info(const std::string &bss_info_str,
+                                  db::bss_info_conf_t &bss_info_conf, std::string &err_string)
+{
+    auto confs = string_utils::str_split(bss_info_str, ' ');
+    if (confs.size() != 8) {
```

Comment:
It is also valid to have just two parameters: ALID and operating class. In that case, the entries for that ALID/operatingclass combination should be removed. Cfr. the second example for BSS_Info1.

Reply:
I believe they had a copy/paste error there. The description says otherwise.
If It appears to be that I wrong, I'll fix it.

Reply:
No, the description says it too (though the wording is a bit confusing): "To clear the BSS info stored for a specific operating class, only the device ALID and operating class are retained."